web/src/main/java/no/nav/ung/sak/web/app/tjenester/saksbehandler/SaksbehandlerRestTjeneste.java

```diff
@@ -55,9 +56,10 @@ public SaksbehandlerRestTjeneste() {
 
     @Inject
     public SaksbehandlerRestTjeneste(
-        @KonfigVerdi(value = "systembruker.username", required = false) String systembruker,
+        MicrosoftGraphTjeneste microsoftGraphTjeneste, @KonfigVerdi(value = "systembruker.username", required = false) String systembruker,
```

Comment:
Gjør vi kallene med systembruker?

Hadde det ikke vært bedre å bruke Azure systemtoken, evt. med OBO?

Reply:
Kallene blir gjort med Azure systemtoken, referansen til systembruker er for å filtrere den bort fra listen av saksbehandlere

Reply:
Aha, nå ser jeg det 👍